website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,200 @@ description: |-
+in Serf.
+
+### Example Server Configuration
+
+The following server agent configuration will create two user-defined network
+segments: `alpha` and `beta`.
+
+<CodeTabs heading="Example network segments configuration for server agents">
+
+```hcl
+segments = [
+  {
+    name = "alpha"
+    bind = "{{GetPrivateIP}}"
+    advertise = "{{GetPrivateIP}}"
+    port = 8303
+  },
+  {
+    name = "beta"
+    bind = "{{GetPrivateIP}}"
+    advertise = "{{GetPrivateIP}}"
+    port = 8304
+  }
+]
+```
+
+```json
+{
+  "segments": [
+    {
+      "name": "alpha",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8303
+    },
+    {
+      "name": "beta",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8304
+    }
+  ]
+}
+```
+
+</CodeTabs>
+
+The server [agent configuration](/docs/agent/options) options relevant to network
+segments are:
+
+- [`ports.serf_lan`](/docs/agent/options#serf_lan_port): The Serf LAN port on this server
+  for the `<default>` network segment's gossip pool.
+- [`segments`](/docs/agent/options#segments): A list of user-defined network segments
+  on this server, including their names and Serf LAN ports.
+
+## Client Configuration
+
+Each client agent is a member of only one segment at a time. This will be the
+`<default>` segment unless otherwise specified in their [`segment`](/docs/agent/options.html#_segment)
+agent configuration option.
+
+### Join a Client to a Segment ((#join_a_client_to_a_segment))
+
+For a client agent to [join](/commands/join) the Consul
+datacenter, it must connect to another agent (client or server) within its
+configured segment.
+
+<Tabs>
+<Tab heading="Join via another client">
+
+Clients A and B specify the same segment S. Client B is already joined to the segment S
+LAN gossip pool. Client A wants to join via Client B. In order to do so, Client A
+must connect to Client B's configured [Serf LAN port](/docs/agent/options#serf_lan_port).
+
+</Tab>
+<Tab heading="Join via a server">
+
+Client A specifies segment S and wants to join the segment S gossip pool via Server 1.
+In order to do so, Client A must connect to Server 1's configured [Serf LAN port
+for segment S](/docs/agent/options#segment_port).
+
+</Tab>
+</Tabs>
+
+There are several methods to specify the port used by the join operation, listed in order
+of precedence:
+
+1. Specify an explicit port in the join address. For example,
+   `consul agent -retry-join "client-b-address:8303"` when starting Client A using the CLI,
+   or in Client A's configuration using the [retry-join option](/docs/agent/options#retry_join)
+   without cloud auto-join. If using `retry-join`
+   [with cloud auto-join](https://www.consul.io/docs/install/cloud-auto-join#auto-join-with-network-segments),
+   there is no method to specify an explicit port to use, so one of the implicit methods
+   below must be used instead.
```

Comment:
This gives the impression that cloud auto-join might be usable when the segment port is defined at the CLI.

```suggestion
1. Specify an explicit port in the join address. For example,
   `consul agent -retry-join "client-b-address:8303"` when starting the agent using the CLI,
   or in the agent's configuration using the [retry-join option](/docs/agent/options#retry_join). This method is not compatible with [cloud auto-join](/docs/install/cloud-auto-join#auto-join-with-network-segments).
```